appveyor.yml

```diff
@@ -1,22 +1,35 @@
+image: Visual Studio 2017
+
 environment:
+  channel: nightly
   matrix:
-  - TARGET: nightly-x86_64-pc-windows-msvc
-  - TARGET: nightly-i686-pc-windows-msvc
-  - TARGET: nightly-x86_64-pc-windows-gnu
-  - TARGET: nightly-i686-pc-windows-gnu
-  - TARGET: 1.6.0-x86_64-pc-windows-gnu
```

Comment:
Please preserve testing of 1.6.0 as it is important for not making breaking changes to the minimum supported Rust version.